src/platform/web/docroot/sw.js

```diff
@@ -69,7 +69,9 @@ async function purgeOldCaches() {
 }
 
 self.addEventListener('fetch', (event) => {
-    event.respondWith(handleRequest(event.request));
+    if (event.request.method === "GET") {
```

Comment:
can we add a comment here that we intercepting uploads here would break progress events?